ChatApp/ChatView/MessageCell.swift

```diff
@@ -0,0 +1,156 @@
+//
+//  MessageCell.swift
+//  ChatApp
+//
+//  Created by Giorgi Makadze on 21.04.2023.
+//
+
+import Foundation
+import UIKit
+
+class MessageCell: UICollectionViewCell {
+    private var leadingConstraints: [NSLayoutConstraint] = []
+    private var trailingConstraints: [NSLayoutConstraint] = []
+    let label = UILabel()
+    let containerView = UIView()
+    //    let elipse = UIView()
+    let textDate = UILabel()
+    //    let label2 = UILabel()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("nope!")
+    }
+    
+    private func setup() {
+        contentView.backgroundColor = .clear
+        
+        makeContainer()
+        makeLabel()
+        makeTextDate()
+        
+        setupContainerViewConstant()
+        setupLabelConstant()
+        setupTextDateConstant()
+    }
+    
+    private func makeContainer() {
+        containerView.backgroundColor = Constants.containerViewBackgroundColor
+        containerView.layer.cornerRadius = CGFloat(Constants.containerViewRadius)
+        contentView.addSubview(containerView)
+    }
+    
+    private func makeTextDate() {
+        textDate.textColor = Constants.textDateTextColor
+        textDate.font = UIFont.systemFont(ofSize: CGFloat(Constants.textDateFontSize))
+        contentView.addSubview(textDate)
+    }
+    
+    private func makeLabel() {
+        label.backgroundColor = .clear
+        label.textColor = .black
+        label.font = UIFont.systemFont(ofSize: CGFloat(Constants.labelFontSize))
```

Comment:
Type inference იხმარე, 
```suggestion
        label.font = .systemFont(ofSize: CGFloat(Constants.labelFontSize))
```